ccmlib/scylla_node.py

```diff
@@ -1785,6 +1832,32 @@ def wait_for_compactions(self,
                                      column_family=column_family,
                                      idle_timeout=idle_timeout)
 
+    def run_scylla_types(self, action: str, key_type: ScyllaKeyType, *values: Tuple[Any], extra_args: Optional[List[Any]]=None):
+        """Invoke scylla-types command, with the specified action and extra_args.
+
+        For more information about scylla-types, see https://opensource.docs.scylladb.com/stable/operating-scylla/admin-tools/scylla-types.html
+
+        Params:
+        * action - The scylla-types operation to run.
+        * key_type - The key type of the values being passed to the tool.
+        * value - The values on which the action is to be applied.
+        * extra_args - Additional command-line arguments to pass to scylla-types action.
+
+        Returns: stdout
+
+        Raises: ToolError if scylla-types returns a non-zero exit code.
+        """
+        if extra_args is None:
+            extra_args = []
+        extra_args.extend(key_type.as_types_args())
+
+        scylla_path = common.join_bin(self.get_path(), BIN_DIR, 'scylla')
+        scylla_types_cmd_with_args = [scylla_path, "types", action] + [str(arg) for arg in extra_args] + ["--"] + [str(value) for value in values]
```

Comment:
`[scylla_path, "types", action, *(str(arg) for arg in extra_args), "--", *(str(value) for value in values)]` or
```
args = [scylla_path, "types", action, *extra_args, "--", *values]
args = [str(value) for value in args]
```
Less lists is always better

Reply:
Ack. I have replaced the code with the second variant. Thanks!